lib/src/navbar/navbar.component.scss

```diff
@@ -42,8 +42,8 @@ $navbar-fixed-shadow: 0px 2px 6px $shadow;
         }
         .brand {
             & > img {
-                max-width: 70%;
-                max-height: 70%;
+                height: 37px;
+                width: auto;
```

Comment:
Good call on making this a change to the style rather than an inline fix @benjanderson.  This fix look okay?  I think a fixed height value should be fine for desktop.  And if we need to change the size of the header for mobile, we'll do that with media queries anyway.